proxy.go

```diff
@@ -91,6 +94,12 @@ func (p *proxy) translate(ev *slack.MessageEvent) bot.Message {
 	user, _ := p.Store.UserByID(ev.User)
 	channel, _ := p.Store.ChannelByID(ev.Channel)
 
+	isDirectMessage := ev.Channel[0] == 'D'
+	exp := regexp.MustCompile("^@?" + p.Name + "\\s")
```

Comment:
I think we can skip the regex and assume that people aren't messaging sambot. I'm fine with not handling that - and it prevents us from having to do too much matching

Reply:
Not sure what you mean, if we assume people are not messaging the bot, we don't need this change at all?